go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/digitalocean/godo
 
-go 1.18
+go 1.21
```

Comment:
FYI this creates a problem for downstream users of this library who want their software to be compatible with multiple versions of Go.
Example: https://github.com/prometheus/prometheus/pull/12964
Since nothing in this package requires Go 1.21, it's unnecesary to have it as a minimum requirement.

Reply:
Thanks for flagging this @bboreham! This was a mistake. We've cut a bug fix release ([v1.104.1](https://github.com/digitalocean/godo/releases/tag/v1.104.1)) dropping this to Go 1.20 and documenting our policy of following the [upstream Go policy](https://go.dev/doc/devel/release#policy) of supporting the two latest major releases.

Reply:
Thanks!